Mehdi_test/mod_2.nit

```diff
@@ -0,0 +1,13 @@
+import date_f
+
+var test  = new Date(2014, 01, 09)
+var test2 = new Date(2016, 01, 09)
+var test3 = new Date(2015, 01, 12)
+var res   = test.date_diff(test2,2)
+
+
+print "ToString :" + test.to_s
+print "Date diff : {res}"
+print "Equale : {test == test2}"
```

Comment:
Why are you using string concat with `+` on the line before and superstrings with `{}` on this line?